src/main/scala/logger/LoggerMacro.scala

```diff
@@ -0,0 +1,53 @@
+package logger
+
+import scala.reflect.macros.blackbox.Context
+
+import org.json4s.JsonAST.JValue
+
+private object LoggerMacro {
+
+  type LoggerContext = Context { type PrefixType = Logger }
+
+  def log1(c: LoggerContext)(event: c.Expr[String]) = {
+    import c.universe._
```

Comment:
oups: http://stackoverflow.com/questions/34144734/how-can-i-get-scala-named-and-default-arguments-to-work-with-macros